europace_security.yaml

```
@@ -30,7 +30,10 @@ flows:
         ## Partner-Rechte lesen
       partner:rechte:schreiben: |
         ## Partner-Rechte verändern
-        
+      partner:zugang:silent-sign-on: |
+        ## Silent-Sign-On
```

```suggestion
        ## Silent-Sign-On erlaubt
```
mit Verb ist es leichter verständlich für den Benutzer